uint16[] centerpoint_ids
---
bool success
uint16[] all_icrs

